core-bundle/tests/Routing/RouteProviderTest.php

```diff
@@ -29,6 +29,15 @@
 
 class RouteProviderTest extends TestCase
 {
+    private $pageModelAutoIncrement = 0;
+
+    protected function tearDown(): void
+    {
+        $this->pageModelAutoIncrement = 0;
+
+        parent::setUp();
```

Comment:
you sure?

Reply:
Thats already fixed in #2879, I need to update this PR.

Reply:
Done in 5271027181ae7f966ebd1e450fa45f3eb5cceba8